Toolkit/ArcGISToolkit/AR/ArcGISARView.swift

```diff
@@ -0,0 +1,682 @@
+//
+// Copyright 2019 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+import ARKit
+import ArcGIS
+
+public class ArcGISARView: UIView {
```

Comment:
Hadn't thought about subclassing.  Would `open` be a better choice here?  Since this is open source, I'm not sure if there's a use case for subclassing...

Reply:
could probably allow subclassing later, but we can't dis-allow it later. I would suggest dis-allow it now and open it up later if necessary

Reply:
So dis-allowing it now means leaving it as `public`, correct?

Reply:
Correct.